### PR TITLE
Add SQL Server connector

### PR DESCRIPTION
**Pull request by sanjay990**

### CODE REVIEW

**Comment by sanjay990** [comment]
@akshatnair @petroav

**Comment by sanjay990** [comment]
@rschlussel : Plz review

File: presto-sqlserver/src/main/java/com/facebook/presto/plugin/sqlserver/SqlServerClient.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    public SqlServerClient(JdbcConnectorId connectorId, BaseJdbcConfig config)
+            throws SQLException
+    {
+        super(connectorId, config, "\"", createSqlDriver());
+    }
+
+    private static Driver createSqlDriver()
+    {
+        try {
+            return (Driver) Class.forName("com.microsoft.sqlserver.jdbc.SQLServerDriver").newInstance();
```
Comment (rschlussel-zz) [inline presto-sqlserver/src/main/java/com/facebook/presto/plugin/sqlserver/SqlServerClient.java:39]: Do we not have this class at compile time?  Why are we using reflection here when the other JDBC Plugins instantiate their drivers directly?

**Review by rschlussel-zz — CHANGES_REQUESTED** [review]
You'll also need to add the sql-server plugin to the plugins section of presto.xml (in the presto-server module) so that it gets packaged up.

File: presto-sqlserver/src/main/java/com/facebook/presto/plugin/sqlserver/SqlServerClient.java (hunk 1 above)
Comment (sanjay990) [reply presto-sqlserver/src/main/java/com/facebook/presto/plugin/sqlserver/SqlServerClient.java:39]: Yes, its possible we don't have this class at compile time. If the user doesn't want SQL Server connector,he won't install SQL Server JDBC jar. For other drivers, we download it from maven central and have by default. For SQL Server, user has to manually download and install the drivers before using this connector.

**Comment by akshatnair** [comment]
Please add tests like we have for postgres and mysql

**Comment by sanjay990** [comment]
@rschlussel - https://github.com/Teradata/presto/pull/389/commits/6dd97afb474b9460ac6e71ee68b46d6328e77b44

**Comment by sanjay990** [comment]
@akshatnair - Yes, I am looking into that. We run PostgreSQL and MySQL as embedded database for tests, it's not possible to do the same with SQL Sever. I am looking into running H2 database with `MS SQL Server Compatibility Mode`

**Comment by sanjay990** [comment]
New PR for SQL Server - https://github.com/Teradata/presto/pull/481

**Pull request closed without merging**